modules/add-contextual-data/tests/test_context_info_db.c

```diff
@@ -427,3 +439,53 @@ ParameterizedTest(struct TestNVPairPrefix *param, add_contextual_data, test_impo
       &param->expected,
       1);
 }
+
+Test(add_contextual_data, test_ignore_case_on)
+{
+  gchar csv_content[] = "LoCaLhOsT,tag1,value1";
+  FILE *fp = fmemopen(csv_content, strlen(csv_content) + 1, "r");
```

Comment:
Let's disable the test when fmemopen is not available.

Reply:
based on this https://github.com/balabit/syslog-ng/pull/1942 it should be already disabled

Reply:
Yes you are right, I overlooked it.